core/settings.py

```diff
@@ -0,0 +1,124 @@
+"""
+Django settings for core project.
+
+Generated by 'django-admin startproject' using Django 5.1.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-lmt=s86+%deh+!!+i93mf6^3u&2yknykdm=bhhlf*jcag4@=uz"
```

Comment:
The `SECRET_KEY` is exposed in the settings file. For security reasons, especially in production, it's crucial to keep the secret key confidential. Consider using environment variables or a separate configuration file to manage sensitive information.